README.md

```diff
@@ -1,66 +1,66 @@
-GitHub know where your Tyr instance runs
+    > Here you need to expose **http** port **8080** as the default application
+    interface where the application runs. Consult the ngrok documentation for the
+    information of how to specify these arguments.
 1. Add a webhook for Tyr
-    - In you test-repo - go to Settings -> Webhooks -> Add webhook
-    - Fill in Payload URL -> *your local exposed IP which is provided by 
-    ngrok + “/pull-request”* (e. g., http://f202cf7c.ngrok.io/pull-request)
-    - Fill in Content type -> application/json
-    - Set *Let me select individual events* and check only *Pull requests* 
+    1. In you test-repo - go to Settings -> Webhooks -> Add webhook
+    1. Fill in Payload URL -> *your local exposed IP which is provided by 
+    ngrok + **“/pull-request”*** (e.g., http://f202cf7c.ngrok.io/pull-request)
+    1. Fill in Content type -> application/json
+    1. Set *Let me select individual events* and check only *Pull requests* 
     and *Pull request review comments*
-    - Leave Active set and click “Add webhook”
-    - Don’t worry about the error - the first JSON differs from what the 
-    subsequent PR JSONs will look like
-1. If you create a PR now (create a new branch based on master for instance) 
-you should already see some output in the server logs
-    - Be careful to which repository and branch you are doing the PR!!! 
-    You need to specify your repository, the point is to try to set up Tyr 
-    on your repository.
-1. However, you will still probably see problems with the custom PR check 
-which the app is trying to send to GitHub as there is a requirement to 
-provide an oauth token to the GitHub API
-1. Go to your profile -> settings (top right) -> Developer Settings -> 
-Personal access tokens
-1. Generate new token
-1. Tick repo:status
-1. Generate token
-1. The last step is to let Tyr know what is the token value
-    - add the token to **config.properties** file (the example is provided in 
-    the `tyr-runner/src/main/resources` folder) and copy this file to the same directory
-    where Tyr is running
-1. Restart Tyr
-1. Rerun (update existing or create new PR) - everything should be working now
+    1. Leave Active set and click “Add webhook”
+    1 Don’t worry about the error - the first JSON differs from what the 
+    subsequent PR JSONs will look like.
+1. Last thing is to edit your format template file.  
+    1. Open `format-template.yaml` which path you've set in a 3rd step.
+    1. Edit value of the variable `repository` so it points to __your__ test-repo.
+    1. Edit value of the variable `statusUrl` so it points to __your__ test-repo or leave it default.
+    1. Now you have to set if you want Tyr to use any CI. 
+       * If NO, remove variable `CI` and all of it's content.
+       * If YES, set CI name, and it's properties. We are using TeamCity CI which properties are defined in the `TeamCityProperties` interface.
```

Comment:
tyr.whitelist.enabled=true should be specified for allowing whitelisting and triggering CI